azavea/modellab-geoprocessing
# Add New Layers

Connects azavea/modellab#76

**To Test**
   * There should now be 20 total layers: the 19 listed [here](https://github.com/azavea/modellab/issues/8) (at time of writing) plus `nlcd-zoomed`.
   * Try uploading each of these files to a running instance of the modellab-geoprocessing code and verify that each can produce a tile.
